src/main/java/pl/targosz/homework/FileKeyword.java

```diff
@@ -0,0 +1,18 @@
+package pl.targosz.homework;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Scanner;
+
+
+public class FileKeyword {
+    public static void main(String[] args) throws IOException {
+        Scanner scanner = new Scanner(System.in);
+        String keyword = scanner.next("Kurs");
```

Comment:
A po co tutaj ten scanner?

Reply:
Jestem takim głąbem czasem z tymi plikami, że to szok, wywaliłem to i magicznie mnie olśniło, że O JEZU TO DZIAŁA BEZ TWORZENIA JAKIEGOŚ SCANERA. poprawiłem i działa.